src/util/uxotutil/uxotutil.go

```diff
@@ -0,0 +1,118 @@
+/*************************************************************************
+   > File Name: skycoin_webrpc.go
+   > Author: ben
+   > Mail: zhiyuan_06@126.com
+   > Created Time: 一 10/16 23:09:41 2017
+************************************************************************/
+package uxotutil
+
+import (
+	"sort"
+	"strings"
+
+	"github.com/spaco/spo/src/util/droplet"
+)
+
+//AccountMgr manager all unspent outputs
+type AccountMgr struct {
```

Comment:
Is this new package necessary? Can put it in daemon or visor or gui package?

Reply:
move to visor